pytablewriter/writer/text/_rst.py

```diff
@@ -119,12 +119,10 @@ def write_table(self, **kwargs) -> None:
         IndentationTextTableWriter.write_table(self, **kwargs)
 
     def _get_opening_row_items(self) -> List[str]:
-        directive = ".. csv-table:: "
```

Comment:
It would be preferable not to delete the variable definition.
How about just remove a trailing white space from the value, like:

```
directive = ".. csv-table::"
```

Reply:
Sure. Fixed.